darwin/future/meta/queries/item.py

```diff
@@ -60,6 +61,14 @@ def _collect(self) -> Dict[int, Item]:
         }
         return items
 
+    def sort(self, **kwargs: str) -> None:
+        sorting_methods = SortingMethods(**kwargs)
```

Comment:
`(**kwargs)` here throws a type warning I've had trouble getting to the bottom of

```
Argument of type "str" cannot be assigned to parameter "accuracy" of type "Literal['asc', 'desc'] | None" in function "__init__"
  Type "str" cannot be assigned to type "Literal['asc', 'desc'] | None"
    Type cannot be assigned to type "None"
    "str" cannot be assigned to type "Literal['asc']"
    "str" cannot be assigned to type "Literal['desc']"Pylance[reportGeneralTypeIssues](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportGeneralTypeIssues)
```

Reply:
Aha, that makes sense, cheers